recipes/pyzbar/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pyzbar" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/NaturalHistoryMuseum/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: "f51c82c2864f8e5a8d44f55853e027f8cbc592324b7afffa62100f2f9c54cbdb"
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
I bet this one can be noarch: python

Reply:
Enabling noarch option throws below error:
[noarch_python] Noarch package contains binary script: read_zbar.exe

Reply:
Why does the source contain a compiled binary?

Reply:
to be clear, the recipe has to build the package from source

Reply:
Added noarch option, the above issue was seen when using wheel.